### DISCUSSION TITLE
[discord] add format for ds messages

### DISCUSSION THREAD

**Discussion by TOWUK** [EssentialsX/Essentials#4801] [Ideas and Feature Suggestions] [answered]
add support messages from server like a chat mc!
![image](https://user-images.githubusercontent.com/28830162/153701729-6b373f1b-c5bf-47f9-afec-7005c0187605.png)
![image](https://user-images.githubusercontent.com/28830162/153701738-18658462-af7b-4e85-ada3-721a6228f517.png)

**Comment by JRoy** [comment, marked as answer]
> add support messages from server like a chat mc!
> ![image](https://user-images.githubusercontent.com/28830162/153701729-6b373f1b-c5bf-47f9-afec-7005c0187605.png)
> ![image](https://user-images.githubusercontent.com/28830162/153701738-18658462-af7b-4e85-ada3-721a6228f517.png)
> 

`>` and `<` are special characters in discord. You have to put a `\` before them in order to have them show up.
